tooling/noir_js/src/input_validation.ts

```diff
@@ -0,0 +1,55 @@
+// Check if all of the input values are correct according to the ABI
+export function validateInputs(inputs, abi) {
+  for (const param of abi.parameters) {
+    const inputValue = inputs[param.name];
+    if (inputValue === undefined) {
+      // This is checked by noirc_abi, so we could likely remove this check
```

Comment:
Should we make an issue here?

Reply:
The code in this file will be migrated to abi, its not needed just provides a bit of a better ux